packages/marketplace/src/services/__tests__/billings.ts

```diff
@@ -1,13 +1,14 @@
 import { fetchMonthlyBilling } from '../billings'
 import { generateHeader, URLS } from '@/constants/api'
-import { fetcher, setQueryParams } from '@reapit/elements'
+import * as Element from '@reapit/elements'
 
 describe('should fetchMonthlyBilling run correctly', () => {
   it('fetcher should be called', () => {
     const applicationId = ['applicationId']
     const month = '2020-05'
     const headers = generateHeader(window.reapit.config.marketplaceApiKey)
-    const url = `${URLS.trafficEvents}/billing/${month}?${setQueryParams({ applicationId })}`
+    const url = `${URLS.trafficEvents}/billing/${month}?${Element.setQueryParams({ applicationId })}`
+    const fetcher = spyOn(Element, 'fetcher')
```

Comment:
You can just `spyOn(window, 'fetch')` as we mock the fetch object on jest.setup. Or, you can just jest.mock('@reapit/elements') and will pick up the fetcher mock in the Elements mock folder

Reply:
I updated it. you can review it again